cdisc_rules_engine/utilities/rule_processor.py

```diff
@@ -249,6 +249,10 @@ def perform_rule_operations(
                 standard_version=standard_version,
                 meddra_path=kwargs.get("meddra_path"),
                 whodrug_path=kwargs.get("whodrug_path"),
+                ct_version=operation.get("version"),
+                ct_attribute=operation.get("attribute"),
+                ct_package=kwargs.get("ct_package"),
+                # ct_package=kwargs.get("codelist_term_maps"),
```

Comment:
Why was this removed? I don't think it should be

Reply:
I don't need the codelist_term_maps to be pre-processed. I process them in the operation operator.

Reply:
That may be true for this operation, but I don't think it is true fro other operators

Reply:
I still think this should be added back, unless this arg is not used by other operations

Reply:
Removed the comment so we will have two variables related to controlled terminologies.